src/Kafka/KafkaInstanceDrawer/components/SampleCodeSelect.tsx

```diff
@@ -0,0 +1,505 @@
+          );
+        case "producer":
+          return (
+            <div>{`package org.acme.kafka;
+
+        import java.time.Duration;
+        import java.util.Random;
+        
+        import javax.enterprise.context.ApplicationScoped;
+        
+        import io.smallrye.mutiny.Multi;
+        import org.eclipse.microprofile.reactive.messaging.Outgoing;
+        
+        /**
+         * A bean producing random prices every 5 seconds.
+         * The prices are written to a Kafka topic (prices). The Kafka configuration is specified in the application configuration.
+         */
+        @ApplicationScoped
+        public class PriceGenerator {
+        
+            private Random random = new Random();
+        
+            @Outgoing("generated-price")
+            public Multi<Integer> generate() {
+                return Multi.createFrom().ticks().every(Duration.ofSeconds(5))
+                        .onOverflow().drop()
+                        .map(tick -> random.nextInt(100));
+            }
+        
+        }`}</div>
+          );
+        case "consumer":
+          return (
+            <div>{`package org.acme.kafka;
+
+        import javax.enterprise.context.ApplicationScoped;
+        
+        import org.eclipse.microprofile.reactive.messaging.Acknowledgment;
+        import org.eclipse.microprofile.reactive.messaging.Incoming;
+        import org.eclipse.microprofile.reactive.messaging.Outgoing;
+        
+        import io.smallrye.reactive.messaging.annotations.Broadcast;
+        
+        /**
+         * A bean consuming data from the "prices" Kafka topic and applying some conversion.
+         * The result is pushed to the "my-data-stream" stream which is an in-memory stream.
+         */
+        @ApplicationScoped
+        public class PriceConverter {
+        
+            private static final double CONVERSION_RATE = 0.88;
+        
+            // Consume from the \`prices\` channel and produce to the \`my-data-stream\` channel
+            @Incoming("prices")
+            @Outgoing("my-data-stream")
+            // Send to all subscribers
+            @Broadcast
+            // Acknowledge the messages before calling this method.
+            @Acknowledgment(Acknowledgment.Strategy.PRE_PROCESSING)
+            public double process(int priceInUsd) {
+                return priceInUsd * CONVERSION_RATE;
+            }
+        
+        }`}</div>
+          );
+        default:
+          return null;
+      }
+    case "springboot":
+      switch (codeSnippet) {
+        case "config":
+          return (
+            <div>{`package com.example.kafkaconfig;
+          import java.util.HashMap;
+         import java.util.Map;
+        
+         public class KafkaConfig {
+        
+             public static Map<String, Object> Config() {
+        
+                 Map<String, Object> config = new HashMap<>();
+        
+                 String kafkaHost = System.getenv("<bootstrap_server>");
```

Comment:
For client ID and client secret, we want to use the angle brackets to communicate to the user that those are values the user needs to specify.

But for this line, we know what the bootstrap URL is for the selected Kafka instance, so we will need to prepopulate that in this sample code. But looking at the current UI preview with the description text, it looks like the user is expected to provide all 3 values. 

@riccardo-forina - since this bootstrap URL will be generated within consoledot based on the selected Kafka, is there another way we should represent this value in the storybook component so that it's more obvious it will be prepopulated? 

The following is an example of what the current UI component looks like.

![image](https://user-images.githubusercontent.com/21063328/212984137-bd372645-944e-43e5-8728-d5c7aaaff2e4.png)